scripts/zones/Oldton_Movalpolos/mobs/Goblin_Wolfman.lua

```diff
@@ -0,0 +1,21 @@
+-----------------------------------
+-- Area: Oldton Movalpolos
+--  MOB: Goblin Wolfman
+-----------------------------------
+require("scripts/zones/Oldton_Movalpolos/MobIDs");
+require("scripts/globals/settings");
```

Comment:
it'll get taken care of later but just so you know, we don't actually have to require settings or mobid's in the wolfman script because he's not using anything form them. he only needed status because thats where dsp.mobMod.IDLE_DESPAWN got defined

Reply:
I was working on a bunch of custom stuff for lol's and it became a habit to put that in XD

Reply:
I planned on making the Scrawled Writing to work as intended of spawning in a random location (3 places iicr). I can make the correction then.

Reply:
we have a utils function for shuffling to diff positions I think.

the way to do it is despawn npc when NM spawns (right in the trade function) and choose a new random pos. retail has it hide right when you spawn him. the random location list includes the current location so sometimes it looks as if it "didn't move".